runtime/bcutil/ROMClassBuilder.hpp

```diff
@@ -109,17 +109,20 @@ class ROMClassBuilder
 
 	
 
-	/* NOTE: Be sure to update J9DbgROMClassBuilder.st when changing the state variables below. */
+	/* NOTE: Be sure to update J9DbgROMClassBuilder in j9nonbuilder.h when changing the state variables below. */
 	J9JavaVM *_javaVM;
 	J9PortLibrary * _portLibrary;
 	U_8 * _verifyExcludeAttribute;
 	VerifyClassFunction _verifyClassFunction;
 	UDATA _classFileParserBufferSize;
 	UDATA _bufferManagerSize;
 	U_8 *_classFileBuffer;
+	U_8 *_anonClassNameBuffer;
+	UDATA _anonClassNameBufferLength;
```

Comment:
@tajila Why is this field call `_anonClassNameBufferLength` instead of `_anonClassNameBufferSize` which would follow the existing pattern? Same comment applies to j9nonbuilder.h.

Reply:
It was a bit of an oversight, I'll make another PR to fix it.

Reply:
Thanks @tajila.